test/mitmproxy/addons/test_savehar.py

```diff
@@ -173,6 +173,19 @@ def test_savehar(log_file: Path, tmp_path: Path, monkeypatch):
     assert actual_har == expected_har
 
 
+def test_flow_entry():
+    s = SaveHar()
+    flow = tflow.twebsocketflow(
+        method="CONNECT", host="test.test", port=443, scheme="https"
+    )
+    servers_seen: set[Server] = set()
+
+    flow_entry = s.flow_entry(flow, servers_seen)
+
+    if flow_entry["request"]["method"] == "CONNECT":
+        assert flow_entry["request"]["url"].startswith("https")
```

Comment:
This if statement feels redundant, as we already know the method will be CONNECT, as per the configuration above. The following should work too:

```suggestion
    assert flow_entry["request"]["url"].startswith("https")
```

Reply:
That makes sense, thanks for the suggestion!